hedyorg/hedy
# [CONTENT] Add `pressed` keyword to cheat sheets

**Description**

This PR adds English cheat sheet entries for the `pressed` keyword at levels 5 and 8.
Will probably conflict with #3795 , please tell me if anything needs changing to make this PR merge more smoothly!

**Fixes #3779 **

**How to test**

* Open Hedy at level 5 or 8 (in English for now)
* Click the cheat sheet / "🤔" button
* An entry for `pressed` should be in the list!
